### DISCUSSION TITLE
Literate-nav: confused about the interplay between README.md and wildcards

### DISCUSSION THREAD

**Discussion by pitosalas** [mkdocs/mkdocs#3610] [Q&A]
One specific example: I would like the top menu to have drop downs based on wildcards. But I also want a home page with text on it. I keep going around in circles trying to understand when wildcards work and when they don't; whether an .md with wildcards should ever be used for actual content (vs. nav) and so on. I have gotten wildcards to work by playing around but I am not clear on the rules.

So basically can anyone give an explanation of exactly where wildcards work, and what they do when they work?

**Comment by pawamoy** [comment]
Could you show us some code, and explain what you expect and what happens instead? That will make it easier to understand what you describe (it's not always easy with just text).

**Comment by pitosalas** [comment]
Sorry, yes. My repo is: https://github.com/campusrover/labnotebook2

Building this gives the error: "Did not find any item/section content
specified.” Which I think is because I have a numbered list in README.md.
Does that mean that my content cannot have a numbered list because it is
interpreted like a literate-nav navigation block?

Now if I remove that numbered list, my second question is, how would I go
about using the wildcard notation to be able to get the top navbar to
include for example, all the .mds in one of my directories? Where would I
put that?

Thanks!
